src/org/jetbrains/java/decompiler/main/extern/IFernflowerPreferences.java

```diff
@@ -382,6 +382,12 @@ public interface IFernflowerPreferences {
   @ShortName("mcs")
   String MARK_CORRESPONDING_SYNTHETICS = "mark-corresponding-synthetics";
 
+  @Name("Excluded Class Regex")
+  @Description("Exclude classes from decompilation if their fully qualified names match the specified regular expression.")
+  @ShortName("exc")
+  @Type(Type.STRING)
+  String EXCLUDED_CLASS_REGEX = "excluded-class-regex";
```

Comment:
Thank you for making it a regex! I just have two small things before merging. For one, I think instead of `excluded-class-regex` the option should be named `exclude-classes`, since the regex is already specified in the description. `@ShortName("exc")` should also be removed since short names are legacy and no longer being added for new options. Thanks a lot!

Reply:
no problem, updated! :)